.github/workflows/release.yml

```diff
@@ -113,3 +113,40 @@ jobs:
         run: echo "Changes exist in validator" && yarn npm publish --access public
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      #Add to github releases
+      - name: Release Cli
```

Comment:
Can you make it `Publish Node` -> `Release Node` -> `Publish Common` -> `Release Common` ...
so that even if there's a step fails, for example `Publish Common`, the release of Node is still posted.